src/Docs/Resources/current/2-internals/1-core/20-data-abstraction-layer/120-translations.md

```diff
@@ -4,7 +4,43 @@ The data abstraction layer supports translations out of the box. In order to do
 
 ### A translation definition?
 
-Breaking these fields into different tables enables you to search and inherit data very easily. It provides a strict structure and data consistency is ensured through constraints.
+Breaking these fields into different tables enables you to search and inherit data very easily.
+It provides a strict structure and data consistency is ensured through constraints.
+Given the following DDL for MySQL:
+
+```sql
+CREATE TABLE `product` (
+    `id` BINARY(16) NOT NULL,
+    `additional_text` VARCHAR(255),
+    `created_at` DATETIME(3) NOT NULL,
+    `updated_at` DATETIME(3) NULL,
+    PRIMARY KEY (`id`)
+);
+```
+
+it will be split up like this to provide the previously listed features:
+
+```sql
+CREATE TABLE `product` (
+    `id` BINARY(16) NOT NULL,
+    `created_at` DATETIME(3) NOT NULL,
+    `updated_at` DATETIME(3) NULL,
+    PRIMARY KEY (`id`)
+);
+CREATE TABLE `product_translation` (
+    `product_id` BINARY(16) NOT NULL,
+    `language_id` BINARY(16) NOT NULL,
+    `additional_text` VARCHAR(255) NOT NULL,
+    `created_at` DATETIME(3) NOT NULL,
+    `updated_at` DATETIME(3),
+    PRIMARY KEY (`product_id`, `language_id`),
+    CONSTRAINT `fk.pnp_ruleset_translation.language_id` FOREIGN KEY (`language_id`)
```

Comment:
btw your constrain name is wrong

Reply:
the PR is already imported. I comment on the internal one

Reply:
Oh no. My private project leaked into it :( Snap. Will you fix it or will I?